### DISCUSSION TITLE
omnibar styling syntax clarification

### DISCUSSION THREAD

**Discussion by mudduck** [gdh1995/vimium-c#1026] [Q&A] [answered]
for the following element how would i add bold emphasis too the .title class in the omnibar? this is what i tried to no avail: 
`/* omni */
#bar > #list > .item > .top > .title { font-weight: bold; }`

thx

**Comment by gdh1995** [comment, marked as answer]
It's `/* #omni */`, while your version lacks the `#` character.

This should work.
```
/* #omni */
.title { font-weight: bold; }
```

**Reply by mudduck** [reply]
ty
